pkgs/by-name/lu/luarocks-packages-updater/package.nix

```diff
@@ -16,52 +12,45 @@ let
     nix nix-prefetch-scripts luarocks-nix
   ];
 
-  luaversions = [
-    lua5_1
-    lua5_2
-    lua5_3
-    lua5_4
-  ];
-
+  attrs = builtins.fromTOML (builtins.readFile ./pyproject.toml);
+  pname = attrs.project.name;
+  inherit (attrs.project) version;
 in
+
 python3Packages.buildPythonApplication {
-  pname = "luarocks-packages-updater";
-  version = "0.1";
+  inherit pname version;
+  pyproject = true;
 
-  format = "other";
+  src = lib.cleanSource ./.;
 
   nativeBuildInputs = [
     makeWrapper
-    python3Packages.wrapPython
+    python3Packages.setuptools
```

Comment:
Technically, this should go in `build-system`  but it is more cosmetic than useful so not a blocker.

Reply:
I've replaced with build-system

Reply:
Thanks ! You have gotten rid of `makeWrapper`. Was that on purpose ?

Reply:
yes it seems to work without, the buildPythonPackage brings it automatically I think

Reply:
Ok cool !